doc/users/fonts.rst

```diff
@@ -121,7 +121,7 @@ This is especially helpful to generate *really lightweight* documents.::
   These core fonts are limited to PDF and PS backends only; they can not be
   rendered in other backends.
 
-  Another downside to this is that while the font metric are standardized,
+  Another downside to this is that while the font metric is standardized,
```

Comment:
```suggestion
  Another downside to this is that while the font metrics are standardized,
```

Agree there is a typo, but Ithink metric should be made plural rather than the verb being made singular?.

Reply:
@tacaswell, I agree with you.